lib/_javascripts/reviewNotify.js

```diff
@@ -0,0 +1,157 @@
+const fs = require('fs');
+const path = require('path');
+const cheerio = require('cheerio');
+// dynamic import required here
+const fetch = (...args) =>
+  import('node-fetch').then(({ default: fetch }) => fetch(...args));
+
+const output = './_site/nhsbsa-digital-playbook';
+const playbookIssueUrl =
+  'https://api.github.com/repos/nhsbsa/nhsbsa-digital-playbook/issues';
+const GITHUB_TOKEN = process.env.GITHUB_TOKEN ?? '';
+const defaultRequestOptions = {
+  method: 'GET',
+  headers: {
+    Authorization: `token ${GITHUB_TOKEN}`,
+  },
+};
+
+/**
+ * The `createGitHubIssue` function is an asynchronous function that creates a new GitHub issue using the GitHub API. It
+ * takes a `title` and `body` as parameters, which represent the title and body of the issue respectively.
+ *
+ * @async
+ * @function
+ * @name createGitHubIssue
+ * @kind function
+ * @param {any} title
+ * @param {any} body
+ * @returns {Promise<void>}
+ */
+async function createGitHubIssue(title, body) {
+  if (!GITHUB_TOKEN) return console.error('No Github token set');
+  try {
+    const response = await fetch(playbookIssueUrl, defaultRequestOptions);
+    const issuesData = await response.json();
+    const existingIssue = issuesData.find((issue) => issue.title === title);
+
+    if (!existingIssue) {
```

Comment:
can it handle when an issue exists, but has been resolved? so we can re-use the same issue